docs/section5/Rule5-49.md

```diff
@@ -0,0 +1,53 @@
+
+        - Check if surface is regulated, add zone total area of building envelope to building total: `if scc_dict_p[surface_p.id] != "UNREGULATED": building_total_envelope_area += sum(surface.area for surface in zone_p.surfaces)`
+
+      - Check if zone is conditioned or semi-heated, add zone air leakage rate to building total: `if zone_conditioning_category_dict_p[zone.id] in [CONDITIONED RESIDENTIAL, CONDITIONED NON-RESIDENTIAL, CONDITIONED MIXED, SEMI-HEATED]: building_total_air_leakage_rate += zone_p.infiltration.air_leakage_rate`
+
+- Calculate the required proposed design air leakage rate at 75Pa: `target_air_leakage_rate_p = 0.6 * building_total_envelope_area * 0.00047194745`
+
+**Rule Assertion:**  
+
+- Case 1: For P_RMR, if infiltration pressure difference is 75Pa and the total air leakage rate for conditioned and semi-heated zones is equal to the required proposed design air leakage rate at 75Pa: `if ( P_RMR.ASHRAE229.infiltration_pressure_difference == 75 ) AND ( building_total_air_leakage_rate == target_air_leakage_rate_p ): PASS`
+
+- Case 2: Else if infiltration pressure difference is 75Pa for P_RMR and the total air leakage rate for conditioned and semi-heated zones is not equal to the required proposed design air leakage rate at 75Pa: `if ( P_RMR.ASHRAE229.infiltration_pressure_difference == 75 ) AND ( building_total_air_leakage_rate != target_air_leakage_rate_p ): FAIL`
+
+- Case 3: Else if infiltration pressure difference is 0Pa for P_RMR and the total air leakage rate for conditioned and semi-heated zones is equal to the required proposed design air leakage rate at 0Pa (0.112 is referenced in G3.1.1.4 Modeling Building Envelope Infiltration): `if ( P_RMR.ASHRAE229.infiltration_pressure_difference == 0 ) AND ( building_total_air_leakage_rate == target_air_leakage_rate_p * 0.112 ): PASS`
+
+- Case 4: Else if infiltration pressure difference is 0Pa for P_RMR and the total air leakage rate for conditioned and semi-heated zones is not equal to the required proposed design air leakage rate at 0Pa (0.112 is referenced in G3.1.1.4 Modeling Building Envelope Infiltration): `if ( P_RMR.ASHRAE229.infiltration_pressure_difference == 0 ) AND ( building_total_air_leakage_rate != target_air_leakage_rate_p * 0.112 ): FAIL`
+
+- Case 5: Else, infiltration pressure difference is neither 75Pa nor 0Pa for P_RMR: `else: FAIL and raise_warning "90.1 SECTION G3.1.1.4 PRESCRIBED PROPOSED DESIGN INFILTRATION RATE AT 75PA AND METHODOLOGY TO CONVERT AIR LEAKAGE RATE AT 75PA TO AIR LEAKAGE AT WIND PRESSURE. THE MODELED INFILTRATION IS AT PRESSURE DIFFERENTIAL OTHER THAN THE TWO OPTIONS PRESCRIBED BY 90.1. EXCEPTION APPLIES WHEN WHOLE-BUILDING AIR LEAKAGE TESTING IS SPECIFIED DURING DESIGN AND COMPLETED AFTER CONSTRUCTION. THE PROPOSED DESIGN AIR LEAKAGE RATE OF THE BUILDING ENVELOPE SHALL BE AS MEASURED. MANUAL CHECK IS REQUIRED TO VERIFY THAT PROPOSED DESIGN INFILTRATION WAS MODELED CORRECTLY."`
+
```

Comment:
To summarize, I think we need the following Cases:
1. PASS when total infiltration_flow_rate equal to target infiltration_flow_rate (no air leakage test was performed). 
2. FAIL when total infiltration_flow_rate not equal to target infiltration_flow_rate but measured air leakage was not entered.
3. PASS or UNDERTERMINED when modeled infiltration is not equal to 0.6 CFM/SF but reflects a measured value correctly converted to infiltration_flow_rate. (UNDERTERMINED outcome would flag for reviewer that completion of air leakage measurements needs to be verified.)
4. FAIL when modeled infiltration is not equal to 0.6 CFM/SF and reflects a measured value incorrectly converted to infiltration_flow_rate. This is a variation of Case 2, so perhaps then can be combined.

Reply:
Updated.